template/node/function/handler.js

```diff
@@ -1,5 +1,33 @@
 "use strict"
 
+// A simple delay function
```

Comment:
What is this for?

Reply:
Will it still work if you reset this back to how it was?

All of the templates in the project are completely minimal example and this might be confusing for people that are used to the template already.